docker/qgisserver/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM camptocamp/qgis-server:3
+LABEL maintainer Camptocamp "info@camptocamp.com"
+
+COPY * /var/www/plugins/geomapfish_qgisserver/
```

Comment:
How is the plugin enabled in QGIS?

Reply:
As I remember QGIS take everything that's in this folder